activesupport/lib/active_support/deprecation/reporting.rb

```diff
@@ -36,6 +36,15 @@ def silence
         @silenced = old_silenced
       end
 
+
+      # Outputs a deprecation warning with 'will be removed' message to
```

Comment:
It's for other AS-using libraries to use, AFAICS (via `gem_name`). Maybe we should provide a more complete usage example along those lines? I agree that no external caller is going to want the "removed from Rails" version.

Maybe try something opening with e.g. "Outputs a deprecation warning for a named method"? What distinguishes this from `warn` is its specific-method focus, to me.

@kaspth Is the symbol "use instead" message popular enough to warrant making it an example, let alone the *only* example? IME, internally, we far more often deprecate a bit more deeply -- not just "use this different name for identical functionality".